.github/workflows/tutorials.yml

```diff
@@ -93,6 +93,13 @@ jobs:
         $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:seis_coverage.xml $SKIP examples/seismic/tutorials/
         $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:acc_coverage.xml $SKIP examples/seismic/acoustic/accuracy.ipynb
 
+    - name: Self-adjoint notebooks
+      run: |
+        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:sa_coverage.xml $SKIP examples/seismic/self_adjoint/
```

Comment:
just `$RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:acc_coverage.xml $SKIP examples/self_adjoint/` should be enough

Reply:
acc_coverage?
why not sa_coverage?

Reply:
No I mean you only need one command not one per notebook.

Reply:
so yes just:
```
- name: Self-adjoint notebooks
run: |
        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:sa_coverage.xml $SKIP examples/seismic/self_adjoint/
```

Reply:
got it